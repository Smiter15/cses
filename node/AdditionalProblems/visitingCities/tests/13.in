204 504
1 2 1
1 2 1
2 3 1
2 3 1
3 4 1
3 4 1
4 5 1
4 5 1
5 6 1
5 6 1
6 7 1
6 7 1
7 8 1
7 8 1
8 9 1
8 9 1
9 10 1
9 10 1
10 11 1
10 11 1
11 12 1
11 12 1
12 13 1
12 13 1
13 14 1
13 14 1
14 15 1
14 15 1
15 16 1
15 16 1
16 17 1
16 17 1
17 18 1
17 18 1
18 19 1
18 19 1
19 20 1
19 20 1
20 21 1
20 21 1
21 22 1
21 22 1
22 23 1
22 23 1
23 24 1
23 24 1
24 25 1
24 25 1
25 26 1
25 26 1
26 27 1
26 27 1
27 28 1
27 28 1
28 29 1
28 29 1
29 30 1
29 30 1
1 30 29
2 30 28
3 30 27
10 30 20
12 30 18
15 30 15
16 30 14
18 30 12
20 30 10
21 30 9
24 30 6
25 30 5
26 30 4
28 30 2
29 30 1
30 31 1
30 31 1
31 32 1
31 32 1
32 33 1
32 33 1
33 34 1
33 34 1
34 35 1
34 35 1
35 36 1
35 36 1
36 37 1
36 37 1
37 38 1
37 38 1
38 39 1
38 39 1
39 40 1
39 40 1
40 41 1
40 41 1
41 42 1
41 42 1
42 43 1
42 43 1
43 44 1
43 44 1
44 45 1
44 45 1
45 46 1
45 46 1
46 47 1
46 47 1
47 48 1
47 48 1
48 49 1
48 49 1
49 50 1
49 50 1
50 51 1
50 51 1
51 52 1
51 52 1
52 53 1
52 53 1
53 54 1
53 54 1
54 55 1
54 55 1
55 56 1
55 56 1
56 57 1
56 57 1
57 58 1
57 58 1
58 59 1
58 59 1
30 59 29
33 59 26
39 59 20
41 59 18
44 59 15
45 59 14
47 59 12
49 59 10
50 59 9
53 59 6
54 59 5
55 59 4
57 59 2
58 59 1
59 60 1
59 60 1
60 61 1
60 61 1
61 62 1
61 62 1
62 63 1
62 63 1
63 64 1
63 64 1
64 65 1
64 65 1
65 66 1
65 66 1
66 67 1
66 67 1
67 68 1
67 68 1
68 69 1
68 69 1
69 70 1
69 70 1
70 71 1
70 71 1
71 72 1
71 72 1
72 73 1
72 73 1
73 74 1
73 74 1
74 75 1
74 75 1
75 76 1
75 76 1
76 77 1
76 77 1
77 78 1
77 78 1
78 79 1
78 79 1
79 80 1
79 80 1
80 81 1
80 81 1
81 82 1
81 82 1
82 83 1
82 83 1
83 84 1
83 84 1
84 85 1
84 85 1
85 86 1
85 86 1
86 87 1
86 87 1
59 87 28
60 87 27
61 87 26
62 87 25
63 87 24
64 87 23
65 87 22
66 87 21
68 87 19
70 87 17
71 87 16
74 87 13
76 87 11
78 87 9
79 87 8
82 87 5
85 87 2
87 88 1
87 88 1
88 89 1
88 89 1
89 90 1
89 90 1
90 91 1
90 91 1
91 92 1
91 92 1
92 93 1
92 93 1
93 94 1
93 94 1
94 95 1
94 95 1
95 96 1
95 96 1
96 97 1
96 97 1
97 98 1
97 98 1
98 99 1
98 99 1
99 100 1
99 100 1
100 101 1
100 101 1
101 102 1
101 102 1
102 103 1
102 103 1
103 104 1
103 104 1
104 105 1
104 105 1
105 106 1
105 106 1
106 107 1
106 107 1
107 108 1
107 108 1
108 109 1
108 109 1
109 110 1
109 110 1
110 111 1
110 111 1
111 112 1
111 112 1
112 113 1
112 113 1
113 114 1
113 114 1
114 115 1
114 115 1
115 116 1
115 116 1
87 116 29
89 116 27
90 116 26
92 116 24
93 116 23
96 116 20
97 116 19
98 116 18
99 116 17
100 116 16
102 116 14
103 116 13
104 116 12
106 116 10
107 116 9
108 116 8
110 116 6
112 116 4
113 116 3
115 116 1
116 117 1
116 117 1
117 118 1
117 118 1
118 119 1
118 119 1
119 120 1
119 120 1
120 121 1
120 121 1
121 122 1
121 122 1
122 123 1
122 123 1
123 124 1
123 124 1
124 125 1
124 125 1
125 126 1
125 126 1
126 127 1
126 127 1
127 128 1
127 128 1
128 129 1
128 129 1
129 130 1
129 130 1
130 131 1
130 131 1
131 132 1
131 132 1
132 133 1
132 133 1
133 134 1
133 134 1
134 135 1
134 135 1
135 136 1
135 136 1
136 137 1
136 137 1
137 138 1
137 138 1
138 139 1
138 139 1
139 140 1
139 140 1
140 141 1
140 141 1
141 142 1
141 142 1
142 143 1
142 143 1
143 144 1
143 144 1
144 145 1
144 145 1
116 145 29
119 145 26
123 145 22
124 145 21
126 145 19
128 145 17
129 145 16
130 145 15
131 145 14
132 145 13
133 145 12
134 145 11
135 145 10
138 145 7
142 145 3
144 145 1
145 146 1
145 146 1
146 147 1
146 147 1
147 148 1
147 148 1
148 149 1
148 149 1
149 150 1
149 150 1
150 151 1
150 151 1
151 152 1
151 152 1
152 153 1
152 153 1
153 154 1
153 154 1
154 155 1
154 155 1
155 156 1
155 156 1
156 157 1
156 157 1
157 158 1
157 158 1
158 159 1
158 159 1
159 160 1
159 160 1
160 161 1
160 161 1
161 162 1
161 162 1
162 163 1
162 163 1
163 164 1
163 164 1
164 165 1
164 165 1
165 166 1
165 166 1
166 167 1
166 167 1
167 168 1
167 168 1
168 169 1
168 169 1
169 170 1
169 170 1
170 171 1
170 171 1
171 172 1
171 172 1
172 173 1
172 173 1
173 174 1
173 174 1
145 174 29
168 174 6
169 174 5
170 174 4
172 174 2
173 174 1
174 175 1
174 175 1
175 176 1
175 176 1
176 177 1
176 177 1
177 178 1
177 178 1
178 179 1
178 179 1
179 180 1
179 180 1
180 181 1
180 181 1
181 182 1
181 182 1
182 183 1
182 183 1
183 184 1
183 184 1
184 185 1
184 185 1
185 186 1
185 186 1
186 187 1
186 187 1
187 188 1
187 188 1
188 189 1
188 189 1
189 190 1
189 190 1
190 191 1
190 191 1
191 192 1
191 192 1
192 193 1
192 193 1
193 194 1
193 194 1
194 195 1
194 195 1
195 196 1
195 196 1
196 197 1
196 197 1
197 198 1
197 198 1
198 199 1
198 199 1
199 200 1
199 200 1
200 201 1
200 201 1
201 202 1
201 202 1
202 204 1
202 204 1
174 204 29
176 204 27
178 204 25
179 204 24
180 204 23
181 204 22
182 204 21
197 204 6
198 204 5
199 204 4
201 204 2
202 204 1
