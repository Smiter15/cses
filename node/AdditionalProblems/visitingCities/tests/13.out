8
1 30 59 87 116 145 174 204 
